### PR TITLE
Fixed delay.md by adding necessary changes in grammars 

### PR DESCRIPTION
**Pull request by akshitagupta15june**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Fixed delay.md by adding necessary changes in grammars 

- **Why this change was needed?** (You can also link to an open issue here)

This change was necessary because some sentences were not grammatically correct
such as "amount of time specified elapses" changed to "specified period of time has elapsed"

Sentences started with "Will determine how long...." to "Regular Delay will determine"

- **Other information**:

### CODE REVIEW

**Comment by akshitagupta15june** [comment]
@scopsy please review

File: docs/docs/platform/delay.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ The delay action awaits a specified amount of time before moving on to trigger t
...
 
-After adding a delay node in the workflow editor, the immediate next step will only be triggered after the amount of time specified elapses.
+After adding a delay node in the workflow editor, the immediate next step will only be triggered after the specified period of time has elapsed.
 
 ### Node configurations
 
 ### Regular Delay
 
-Will determine how long the wait will be before triggering the next step.
+Regular Delay will determine how long will it take before triggering the next step.
```
Comment (davidsoderberg) [inline docs/docs/platform/delay.md:17]: how it will take?

Comment (akshitagupta15june) [reply docs/docs/platform/delay.md:17]: ok adding changes

Comment (akshitagupta15june) [reply docs/docs/platform/delay.md]: @davidsoderberg done

**Review by davidsoderberg — APPROVED** [review]

**Pull request merged**